javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -1641,4 +1644,25 @@ public static boolean isWideningBinary(BinaryTree node) {
             }
         }
     }
+
+    /**
+     * Creates a LiteralTree for the given value.
+     *
+     * @param typeTag the literal's type tag
+     * @param value a wrapped primitive, null, or a String
+     * @param typeMirror the typeMirror for the literal
+     * @param processingEnv the processing environment
+     * @return a LiteralTree for the given type tag and value
+     */
+    public static LiteralTree createLiteral(
+            TypeTag typeTag,
+            Object value,
+            TypeMirror typeMirror,
+            ProcessingEnvironment processingEnv) {
+        Context context = ((JavacProcessingEnvironment) processingEnv).getContext();
+        TreeMaker maker = TreeMaker.instance(context);
+        LiteralTree result = maker.Literal(TypeTag.BOT, null);
```

Comment:
` maker.Literal(TypeTag.BOT, null)` should be ` maker.Literal(typeTag, value)`, right?  Why doesn't `maker.Literal(typeTag, value)` work? I don't think there are any test for this, so I can't check myself.